Misc/NEWS.d/next/Library/2017-10-17-20-08-19.bpo-31800.foOSCi.rst

```diff
@@ -0,0 +1,3 @@
+Add support for parsing RFC-822/ISO 8601 UTC offsets. strptime '%z' can now
```

Comment:
I don't think the RFC or the ISO standard allows seconds, let alone microseconds in the UTC offset.  Please note that RFC 822 was obsoleted by RFC 2822 which in turn was obsoleted by 5322.  I don't think this matters much for this PR because the RFCs specify the TZ format **without** a colon.  I would remove references to standards and just describe what is implemented.